### PR TITLE
[WIP]Adds in AED's

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
NOTE:
This PR is based of of my other defib PR. So if that one is merged then this one can get merged.
This was needed due to the shitfest it would create otherwise

Adds in AED's (automatic external defibrillator). And their belonging AED mounts.
An AED is used to stop a heart attack. The starting AED's are single use before their battery needs to recharge.

They fit around the belt or the back slot and can be upgraded with a better battery.
Emagged they will behave the same as normal defibs except that they still can't revive people.

Sprites by @Agameofscones 

TODO
- Make the batteries normal ones at roundstart. No free stunprod.
- Do map edits

**Images of sprite/map changes (IF APPLICABLE):**
image
image
image
image
image

**Changelog:**
:cl:
add: Added AED's
/:cl:

### CODE REVIEW

File: code/game/objects/items/aed.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	overlays.Cut()
+	if(!on)
+		overlays += "[initial(icon_state)]-paddles"
+
+/obj/item/twohanded/shockpaddles/aed
+	name = "AED paddles"
+	desc = "A pair of plastic-gripped paddles with flat metal surfaces that are used to deliver electric shocks."
+	icon_state = "defibpaddles0"
+	item_state = "defibpaddles0"
+	revivecost = 10000 // Fucking expensive piece of shit
```
Comment (farie82) [inline code/game/objects/items/aed.dm]: Needs changing to use a normal battery

**Comment by Mitchs98** [comment]
Is this going to replace the doctors current handheld defibs, or is it meant to be a thing alongside it?

**Comment by farie82** [comment]
Along side. This one will be spread around the station for the crew to use in case of emergency

**Comment by Mitchs98** [comment]
Ah good. Just asking because I was approached to code the same thing xD. Cool beans.

**Comment by Fox-McCloud** [comment]
>Along side. This one will be spread around the station for the crew to use in case of emergency

I don't like this, at all. One of the reasons that defibs are rare-ish is that it actually encourages you to have to go to medbay for thing and not just self-treat everything.

Even with old-crit, I would have been opposed to these things; medbay *does* need a niche afterall.

**Comment by KasparoVy** [comment]
I wonder just how much back and forth there's gonna' be about meaningful danger and death in space.

**Comment by Ty-Omaha** [comment]
AED should function like an actual AED and not a portable in game defib

Needs AED voice prompts

"place pads on bare chest"

"no rhythm detected. start compressions"

"pause compressions"

"analyzing rhythm. do not touch the patient"

"shock advised"

"stand clear of patient"

chargeingsound.ogg

ZAP.ogg

"anaylzing rhythm. do not touch the patient"

**Comment by farie82** [comment]
> AED should function like an actual AED and not a portable in game defib, maybe single use for balance too?
> 
> Needs AED voice prompts
> 
> "place pads on bare chest"
> 
> "no rhythm detected. start compressions"
> 
> "pause compressions"
> 
> "analyzing rhythm. do not touch the patient"
> 
> "shock advised"
> 
> "stand clear of patient"
> 
> chargeingsound.ogg
> 
> ZAP.ogg
> 
> "anaylzing rhythm. do not touch the patient"

Currently I have it set that the battery lasts 1 charge. Upgraded ones will have more charges then but who will do that. I'll take a look at the AED's in real life as well

> I don't like this, at all. One of the reasons that defibs are rare-ish is that it actually encourages you to have to go to medbay for thing and not just self-treat everything.
> 
> Even with old-crit, I would have been opposed to these things; medbay _does_ need a niche afterall.

can't self threat yourself here. Try defibbing yourself with a heart attack. Kinda impossible when you're screaming on the ground.
You still need to fix the underlying problems as well. This will just prolong the suffering of the patient.

**Comment by Agameofscones** [comment]
@farie82 Hey Farie, it'd be great if I could get credit for the sprites I handed off to Spartan originally. Cheers, best of luck with the PR.

**Comment by farie82** [comment]
> Hey @farie82 , it'd be great if I could get credit for the sprites I handed off to Spartan originally. Cheers, best of luck with the PR.

Heya, sorry for not including it. I didn't know who made them in all honesty :P 
I've added it now

**Comment by variableundefined** [comment]
I apologize for the late reply, but I agree with Fox that this dilute the value of defibs in Medbay and make it a bit easier to resusciate.

**Comment by variableundefined** [comment]
Closing for reason above

**Pull request closed without merging**